Code/MinimalLib/tests/tests.js

```diff
@@ -1107,6 +1107,144 @@ function test_get_frags() {
     }
 }
 
+function test_hs_in_place() {
+    {
+        var mol = RDKitModule.get_mol("CC");
+        assert(!mol.has_coords());
+        var descNoH = JSON.parse(mol.get_descriptors());
```

Comment:
I'd really prefer something more direct than this.
How about just generating the SMILES and using that?

Reply:
Done.